HW_1/task4.py

```diff
@@ -0,0 +1,12 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+#Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+#а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+number = int(input("введите количество журавликов: "))
+Pet = int(number/6)
```

Comment:
А чем оператор целочисленного отделения плох?
```suggestion
Pet = number // 6
```